documentation/staging/content/userguide/aks/_index.md

```diff
@@ -77,6 +77,7 @@ In this section, you can configure some options about the AKS which will run Web
 | Field | Description |
 |-------|-------------|
 |Create a new AKS cluster| If set to **Yes**, the deployment will create a new AKS cluster resource in the specified resource group. If set to **No**, you have the opportunity to select an existing AKS cluster, into which the deployment is configured. Note: the offer assumes the existing AKS cluster has no WebLogic related deployments. |
+|Use latest supported AKS Kubernetes version| The currently supported version is **1.21.9**.  Oracle tracks the AKS release versions in [Supported Kubernetes versions in Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions). After a new version emerges, Oracle qualifies WLS on AKS against that version and will update the offer to that version. The set of valid version values is listed in [Kubernetes version support policy](https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions#kubernetes-version-support-policy)|
```

Comment:
Fine, except needs a period at the end of the last sentence.

Reply:
Fixed.